doc/classes/Color.xml

```diff
@@ -51,6 +51,26 @@
 				[/codeblocks]
 			</description>
 		</method>
+		<method name="Color" qualifiers="constructor">
+			<return type="Color">
+			</return>
+			<argument index="0" name="code" type="String">
+			</argument>
+			<argument index="1" name="alpha" type="float">
+			</argument>
+			<description>
+				Constructs a [Color] either from an HTML color code or from a standardized color name, with [code]alpha[/code] on the range of 0 to 1. Supported color names are the same as the constants.
```

Comment:
It should be clarified that alpha is overwritten even if specified in the first parameter.